tests/serverTestZeroMQ-DoocsVariableConfig.xml

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <device_server xmlns="https://github.com/ChimeraTK/ControlSystemAdapter-DoocsAdapter">
-  <!-- We use a type which is processed by the ReferenceTestApplication before the data properties used by the test.
-       Otherwise the macro pulse number gets attachted to the next value. The order is determined by the order of types
-       in the HolderMap, as this map is iterated. -->
-  <macro_pulse_number_source>/INT/FROM_DEVICE_SCALAR</macro_pulse_number_source>
+  <!-- It must be irrelevant whether the macro pulse number is processed before or after the variable which is tested by value;
+     the test cases cover both cases. -->
+  <data_matching>exact</data_matching>
+  <macro_pulse_number_source>/UINT/FROM_DEVICE_SCALAR</macro_pulse_number_source>
```

Comment:
Ok, in the test a comment says you want uint for the MPN and int for the data, but does it matter?

Reply:
The reason was that DoocsAdapter supports not all base types - only int but not unsigned int (although DOOCS does support unsigned int already now). The test is a bit easier to write (needs no conversion functions) if only DoocsAdapter-supported base types are tested directly.  Macropulse number then must use a different data type, since each data type is present only once in ReferenceApplication.